DataSourceVMwareGuestInfo.py

```diff
@@ -106,7 +106,8 @@ def get_data(self):
         that the get_data functions in newer versions of cloud-init do,
         such as calling persist_instance_data.
         """
-        if not get_data_access_method():
+        data_access_method = get_data_access_method()
+        if not data_access_method:
             LOG.error("vmware-rpctool is required to fetch guestinfo value")
```

Comment:
Thanks @akutz for this patch. This comment might not related to this patch. Since data_access_method can be VMX_GUESTINFO, /path/to/vmware-rpctool, or None,  then "vmware-rpctool" in the log msg seems not accurate.

Reply:
Yeah, this isn't directly related. And since `VMX_GUESTINFO` is more for testing, I don't feel it's necessary to adjust the happy-path log messages to account for the env var configuration.